Rakefile

```diff
@@ -1,5 +1,10 @@
 require "tmpdir"
 
+# TODO(awong): Add method for overriding these if building on the local
+# machine. Currently it keys off of Travis configs only which makes it
+# less natural as an "API" for local development.
+enable_dev_features = (ENV['TRAVIS_BRANCH'] == "production")
```

Comment:
Looks to me like the logic here is "enable the development features if the branch being merged to is production"  But don't we only want development features when it is **not** production?